horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/TestServiceImpl.java

```diff
@@ -174,7 +167,7 @@ protected boolean checkTestExists(int id) {
     }
 
     @WithRoles(extras = Roles.HORREUM_SYSTEM)
```

Comment:
I think this is no longer required and we can remove it. Elevated privileges here is a security risk.

Reply:
I get errors when I remove it so let's leave it for now.